src/InertSelect.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
why `* as`?

Reply:
Because `React.Element` wasn't a valid Flow type otherwise. The `Element` type is exported separately, not as a property of the main `React` export. I initially wrote this as `import React, { type Element } from 'react'`, but that caused other issues with the gulp build pipeline here that I didn't really feel like fighting haha.

This will admittedly give us a warning message in the SG repo until we upgrade to React 16 there, but it's perfectly safe in both React 15 and 16 since all we're using in this file is the top level `React` and the `React.Element` type.

Reply:
In the past, I've seen React warnings as a result of this. For example, if you're using React 15.6 and do `import * as React`, then `createClass` and `PropTypes` get imported and React throws up a warning

Reply:
Yeah, that warning is strictly in React 15 because those props are removed from `React` in version 16. Since we're not actually using those exports here (we're already using the separate `create-react-class` and `prop-types` packages), it's safe to ignore the warning. When we switch to React 16, it'll go away and everything will continue to work just fine.

Reply:
sounds good to me, 🚢 :shipit: 🚀 📦